rules/downloads.smk

```diff
@@ -1,21 +1,24 @@
-rule download_ensembl_references:
-    output:
-        gfa="data/ensembl/Homo_sapiens.GRCh38.dna.primary_assembly.fa",
-        gff="data/ensembl/Homo_sapiens.GRCh38.81.gff3",
-        pfa="data/ensembl/Homo_sapiens.GRCh38.pep.all.fa",
-        vcf="data/ensembl/common_all_20170710.vcf",
-        vcfidx="data/ensembl/common_all_20170710.vcf.idx"
-    log: "data/ensembl/downloads.log"
-    shell:
-        "(wget -O - ftp://ftp.ensembl.org/pub/release-81//fasta/homo_sapiens/dna/Homo_sapiens.GRCh38.dna.primary_assembly.fa.gz | "
-        "gunzip -c > {output.gfa} && "
-        "wget -O - ftp://ftp.ensembl.org/pub/release-81/gff3/homo_sapiens/Homo_sapiens.GRCh38.81.gff3.gz | "
-        "gunzip -c > {output.gff} && "
-        "wget -O - ftp://ftp.ensembl.org/pub/release-81//fasta/homo_sapiens/pep/Homo_sapiens.GRCh38.pep.all.fa.gz | "
-        "gunzip -c > {output.pfa} && "
-        "wget -O - ftp://ftp.ncbi.nih.gov/snp/organisms/human_9606_b150_GRCh38p7/VCF/GATK/common_all_20170710.vcf.gz | "
-        "gunzip -c > {output.vcf} && "
-        "gatk IndexFeatureFile -F {output.vcf}) 2> {log}"
+# rule download_ensembl_references:
```

Comment:
Why is this commented out?

Reply:
the `unzip_ensembl` is built off `download_ensembl_references` so we don't need it anymore

what `unzip_ensembl` does is:
- if the ensembl files exist, unzip
- else, download references